### PR TITLE
frontend/bidding

### PR DESCRIPTION
**Pull request by veropossumus**

### CODE REVIEW

File: frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            </Heading>
+        </div>
+
+        <div class="mr-5">
+            <Button on:click={handleClick}>
+                {biddingOngoing ? "Start bidding" : "Stop bidding"}
+            </Button>
+
+            <Button disabled={!biddingOngoing}
+                    href="../assignment"
```
Comment (ToastOfToastyness) [inline frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte:62]: when a button is assigned a href, clicking on the button will lead to that page regardless if the button is disabled or not

File: frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        href="/categories/{p.category.id}/{p.id}"
+                        paper={p}
+                        rating={rating}
+                />
+            {/if}
+        {/each}
+    </Papers>
+
+    <div class="mx-auto m-8">
+        <Pagination icon on:next={next} on:previous={previous} {pages}>
```
Comment (ToastOfToastyness) [inline frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte:83]: Are you sure we would need a pagination?

**Review by ToastOfToastyness — CHANGES_REQUESTED** [review]
Not sure about the rating being stored in the paper
But looks good

File: frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte (hunk 2 above)
Comment (veropossumus) [reply frontend/src/routes/categories/[category_id]/bidding/[bidding_id]/+page.svelte:83]: We have pagination in all categories, so I would keep it for the corresponding biddings as well.

**Review by ToastOfToastyness — DISMISSED** [review]

File: frontend/src/routes/categories/[category_id]/+page.svelte [hunk 3]
```diff
@@ -38,19 +48,61 @@
...
+
+
+        {#if mocks[data.category_id - 1].type === "External"}
+            {#if mocks[data.category_id - 1].is_my_category()}
+                <Button size="lg"
+                        color="primary"
+                        class="mx-auto lg:m-0"
+
+                        on:click={mocks[data.category_id - 1].open ? (() => show_assign_modal = true) : ("")}
+                        href={(mocks[data.category_id - 1].open) ?
```
Comment (ToastOfToastyness) [inline frontend/src/routes/categories/[category_id]/+page.svelte:67]: if the category is open, the ExternalAssignModal is opened before then going to the href link ( so it flashed shortly)

**Review by ToastOfToastyness — CHANGES_REQUESTED** [review]
Besides the Modal shorly poping up,
and FR370 (deleting a bidding proccess) missing 
It looks good

**Review by ToastOfToastyness — APPROVED** [review]

**Pull request merged**